mama/c_cpp/src/c/subscription.c

```diff
@@ -452,6 +452,7 @@ mamaSubscription_setupBasic (
     }
 
 #ifdef WITH_ENTITLEMENTS
+	{
```

Comment:
Again, please remove all these WITH_ENTITLEMENTS parenthesis changes for this PR